client/src/components/Visualiser/commandsInputRules.ts

```diff
@@ -20,24 +20,6 @@
 /* -------------------------------------------------------------------------- */
 /*                              Terminal Commands                             */
 /* -------------------------------------------------------------------------- */
-export const getVisualiserTerminalCommands = (
-  topicTitle: string,
-  processCommand: (command: string, args: string[]) => string
-) => {
-  let terminalCommands = {};
-  switch (topicTitle) {
-    case 'Linked Lists':
-      terminalCommands = getLinkedListTerminalCommands(processCommand);
-      break;
-    case 'Binary Search Trees':
-      terminalCommands = getBstTerminalCommands(processCommand);
-      break;
-    default:
-      console.error(`Can't find the terminal commands for ${topicTitle}`);
-  }
-  return terminalCommands;
-};
-
 /* -------------------------- Linked List Commands -------------------------- */
 const getLinkedListTerminalCommands = (processCommand) => ({
```

Comment:
Thought: could we have these files under visualiser-src. It'd be less friction between editing the commands on the terminal and creating visualiser animations. This will also be more scaleable as we increase the number of visualisers.

Reply:
Yeah that sounds like a good idea. How would it be more scalable?

Reply:
This could be good. The idea for having them separated initially was since I thought this was UI-specific and didn't belong in the visualiser source.

One bad thing is that the terminal is quite coupled with these objects. It may make more sense to keep them together for that reason.

Reply:
> Yeah that sounds like a good idea. How would it be more scalable?

We wouldn't have one file with 50 visualiser documentation objects